server/src/test/java/io/seata/server/lock/LockManagerTest.java

```diff
@@ -123,12 +164,25 @@ public void isLockableTest(BranchSession branchSession) throws Exception {
         branchSession2.setBranchId(2L);
         branchSession2.setClientId("c1");
         branchSession2.setResourceGroupId("my_test_tx_group");
-        branchSession2.setResourceId("tb_1");
-        branchSession2.setLockKey("t:1,2");
+        branchSession2.setResourceId(resource);
+        branchSession2.setLockKey(lockKey2);
         branchSession2.setBranchType(BranchType.AT);
         branchSession2.setApplicationData("{\"data\":\"test\"}");
         branchSession2.setBranchType(BranchType.AT);
         return Stream.of(
-                Arguments.of(branchSession1, branchSession2));
+            Arguments.of(branchSession1, branchSession2));
```

Comment:
Please keep the indentation same as other functions.

Reply:
OK